packit_service/worker/handlers/forges.py

```diff
@@ -117,11 +117,14 @@ def run(self) -> TaskResults:
                     f"{'-stg' if self.service_config.deployment == Deployment.stg else ''}. "
                     "We are now onboarding Fedora contributors who have a valid "
                     "[Fedora Account System](https://fedoraproject.org/wiki/Account_System) "
-                    "account. \n\nIf you have such an account, please set the `GitHub Username`"
+                    "account. \n\nYour GitHub username does not match a FAS account "
+                    "username, or you currently don't have the `GitHub Username` field set in your "
+                    "FAS account or your profile is private. "
+                    "Please, set the `GitHub Username`"
                     " field in the settings of the FAS account (if you don't have it set already)"
```

Comment:
I find it difficult to read this without some sort of connector. But maybe it is just me.
And I would not repeat the "field set" thing.

```suggestion
                    "account. \n\nIf your GitHub username does not match the FAS account "
                    "username, or you currently don't have the `GitHub Username` field set in your "
                    "FAS account or your profile is private. "
                    "Then, please, set the `GitHub Username`"
                    " field in the settings of the FAS account."
```

Reply:
the point made in the mentioned issue was that if the issue is opened, it means that one of these is true (accounts doesn't match/the field not set/account private), therefore the `If` is redundant.

Would adding `However` in the beginning of the sentence help?

Reply:
> I find it difficult to read this without some sort of connector. But maybe it is just me.

I had the same. It's much better looking at the resulting file: https://github.com/lbarcziova/packit-service/blob/installation-instructions-fix/packit_service/worker/handlers/forges.py

Reply:
I think it's fine as it is, just two nitpicks:

```suggestion
                    "account. \n\nYour GitHub username does not match the FAS account "
                    "username, or you currently don't have the `GitHub Username` field set in your "
                    "FAS account, or your profile is private. "
                    "Please, set the `GitHub Username`"
                    " field in the settings of the FAS account (if you don't have it set already)"
```

Reply:
I also don't have anything against adding `However`, if it helps.